blogpost-drools-model-compiler.md

```diff
@@ -0,0 +1,116 @@
+# How to start contributing to Drools Executable Model
+
+## What is the Executable Model?
+
+The Executable Model is a new way to execute business rules in Drools which provides a much faster startup time. You can check out the details in the [Drools Documentation](https://docs.jboss.org/drools/release/7.55.0.Final/drools-docs/html_single/index.html) or in other blog posts such as [Mario's](https://blog.kie.org/2021/06/profiling-and-optimizing-executable-model-generation.html).
+
+KJARs built with the `kie-maven-plugin` have the Executable Model enabled since 7.33.0.Final by default and it's the main technology underneath [Kogito](https://kogito.kie.org). 
+With the "Executable Model Compiler", a module you can find in the  `drools-model-compiler` directory, DRL files are transformed into a Java DSL.
+
+## How to start contributing
+
+Drools is a really big open source project, and finding the best way to contribute to it might not be easy.
+Luckily, the Executable Model Compiler is a good way to start, for various reasons:
+
+* It's a fairly new project (as today it's been more or less three years since the inception)
+* It doesn't require deep understanding of the Drools' internal algorithm, PHREAK
+* There always is a former counterpart to verify the code against 
+
+Regarding the third point, we want to make sure that Drools is behaving in the exact same way while using the former runtime (also called "DRL") and the new one (called `PatternDSL`).
```

Comment:
I'd avoid calling it PatternDSL, mostly because the flow one doesn't exist anymore. Let's call it just the executable model DSL.